string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,39 @@
+/* exported titleCase */
+
+function titleCase(title) {
+
+  var i = 0;
+  var singleWord = '';
+  var fullString = '';
+
+  title = title.toLowerCase();
+  title += ' ';
+
+  for (i; i < title.length; i++) {
+
+    while (title[i] !== ' ') {
+
+      if (i === 0 || title[i - 1] === ' ' || singleWord === 'Java' || singleWord === ' Java' || title[i - 1] === '-') {
+        singleWord += title[i].toUpperCase();
+      } else {
+        singleWord += title[i];
+      }
+      i++;
```

Comment:
Rather than make assumptions about the work `'Java'`, it's ok to treat any occurrence of the word `'javascript'` as special and just needs to be re-cased to `'JavaScript'`.